blogposts/more-intuitive-code-search-on-sourcegraph-dot-com.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Search the top 10,000 open source repositories on Sourcegraph.com"
+description: "How Sourcegraph.com became easier to use, by setting the default search scope to the top ten thousand open source repositories on GitHub."
+author: Issac Trotts, Software Engineer at Sourcegraph
+publishDate: 2019-08-16T00:00-11:55
+tags: [
+ blog
+]
+slug: Search-the-top-10000-open-source-repositories-Sourcegraph-com
+heroImage: /blog/sourcegraph-com-usage.png
+published: true
+---
+
+<p style="text-align: center">
+ <img src="/blog/sourcegraph-com-usage.png" />
+ <br/><small>Sourcegraph.com stats as of August 2019</small>
+</p>
+
+Wouldn't it be awesome if you could scope a code search to find usage examples from the top ten thousand open source repositories on GitHub? You can, using [Sourcegraph.com](https://sourcegraph.com), and we've now made it easier to use!
```

Comment:
What do you think about making this a declarative statement that summarizes the post?

```suggestion
[Sourcegraph.com](https://sourcegraph.com) can now search across the top 10,000 open source repositories on GitHub so you can find usage examples of popular projects.
```

Reply:
I don't have a strong opinion here and your version reads better so sounds good!